### PR TITLE
GEN-2777 / Add tagline to ProductHero

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes
Add tagline to ProductHero

Note that we should not set default text alignment in in Heading since we cant make it inherit value from a wrapper then. The browser default is `start` so it will still default to left if no align value is passed

Potential improvement, add as a field in storybook on price calculator page if it should differ from tagline from product metadata 

Screenshot 2024-10-29 at 17 19 40

Screenshot 2024-10-29 at 17 19 48

## Justify why they are needed
Match design

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
* **#4919** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4919?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @gustaveen and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by alebedev — APPROVED** [review]
Looks good

File: packages/ui/src/components/Heading/Heading.tsx [hunk 1]
```diff
@@ -35,7 +35,7 @@ export function Heading({
   className,
   color = 'textPrimary',
   variant = 'standard.32',
-  align = 'left',
+  align,
```
Comment (guilhermespopolin) [inline packages/ui/src/components/Heading/Heading.tsx:38]: Agreed. We just need to make sure this doesn't break uses cases that was relying on this default setting.

**Review by guilhermespopolin — APPROVED** [review]
Have changed that file recently by adding `DiscountTooltip`. I think it should be a simple conflict resolution but ping me if that proves to be wrong :)

File: packages/ui/src/components/Heading/Heading.tsx (hunk 1 above)
Comment (gustaveen) [reply packages/ui/src/components/Heading/Heading.tsx:38]: For sure! I looked around on the site to try to find cases, looked fine. But might have missed some so let's keep eyes open for that 👍

**Pull request merged**